### PR TITLE
Test for PR

### PR DESCRIPTION
**Pull request by vramirez209**
Testing a PR. Updated the specification ID README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ Product specifications will be broken down in many levels as each specification
...
+| PS1001 - Customer/Entity Records | Customer information needs to be captured. Customers can be business, people or portfolios. People have relationships with people, businesses and portfolios. Relationships need to be tracked and can be recursive. For the | Fintechs need to capture FBO (For Beneficiary Owner) records. Similar to Bank Story however, may need a UUID to join to a master relationship. |
+| PS1002 - Products                | Banks offer many products and services. The application should support managing the products and services offered while enabling implementing business logic. Products define the parameters available to accounts.          | Products differ here. These are an api abstraction of the Bank Story                                                                           |
```
Comment (chaiwa-berian) [inline README.md:32]: We do not these details in the Readme file. So I would reject this PR. The specifications will live in its own document and then we can provide a hyperlink in Readme. So  I will close this PR. @alyhoop , waiting for your comment.

**Review by chaiwa-berian — CHANGES_REQUESTED** [review]

**Comment by vramirez209** [comment]
Reopening PR.

**Comment by chaiwa-berian** [comment]
@vramirez209 Kindly remove all the text from the Readme, we will update it later with links. Just leave the title.

**Comment by vramirez209** [comment]
Removed text.

**Review by chaiwa-berian — CHANGES_REQUESTED** [review]
This looks good, but I am not sure why it is failing the CodeQL Analysis Checks. Can you add some python script and re-commit?

**Review by chaiwa-berian — APPROVED** [review]
Great stuff. This looks awesome! I will wait for @alyhoop to review.

**Review by alyhoop — APPROVED** [review]

**Pull request merged**